contentcuration/contentcuration/frontend/channelEdit/components/ResourcePanel.vue

```diff
@@ -127,10 +125,21 @@
               {{ $formatNumber(assessmentItems.length) }}
             </span>
           </DetailsRow>
+          <DetailsRow :label="$tr('description')" :text="getText('description')" />
           <DetailsRow
-            v-if="isExercise"
-            :label="$tr('masteryCriteria')"
-          >
+            v-if="!isTopic"
+            :label="translateMetadataString('level')"
+            :text="level(node.grade_levels)"
+            notranslate
```

Comment:
Rather for my understanding. I assume that categories, learning activities and grade levels are defined by constants and are not user-generated data so I can't understand very well why there's the need to use `notranslate` attribute, could you please elaborate?